### PR TITLE
fix formatting #15126

### PR DESCRIPTION
**Pull request by JohnGaltt**
Fixed formatting due to https://github.com/dotnet/samples/tree/master/snippets/csharp/pipelines per #14414 (comment)

Fixes https://github.com/dotnet/docs/issues/15126

### CODE REVIEW

**Comment by Potapy4** [comment]
Fixes dotnet/docs#15126

**Comment by Rick-Anderson** [comment]
@mairaw  is there anyway to see what the build looks like?

**Comment by JohnGaltt** [comment]
@Rick-Anderson is there a way to build it locally?

**Comment by mairaw** [comment]
Not that I'm aware @Rick-Anderson. I think there is a way to build locally but I never got it to work. 

Let's merge and check?

File: snippets/csharp/pipelines/Program.cs [hunk 1]
```diff
@@ -15,14 +15,16 @@ public static void Main(string[] args)
             var readScheduler = new SingleThreadPipeScheduler();
 
             // Tell the Pipe what schedulers to use and disable the SynchronizationContext.
-            var options = new PipeOptions(readerScheduler: readScheduler, writerScheduler: writeScheduler, useSynchronizationContext: false);
+            var options = new PipeOptions(readerScheduler: readScheduler,
+             writerScheduler: writeScheduler, useSynchronizationContext: false);
```
Comment (Rick-Anderson) [inline snippets/csharp/pipelines/Program.cs]: Params need to be left aligned

File: snippets/csharp/pipelines/ProcessLinesAsync.cs [hunk 2]
```diff
@@ -47,7 +47,8 @@ async Task ProcessLinesAsync(NetworkStream stream)
                 do
                 {
                     // Look for a EOL in the buffered data.
-                    linePosition = Array.IndexOf(buffer, (byte)'\n', bytesConsumed, bytesBuffered - bytesConsumed);
+                    linePosition = Array.IndexOf(buffer, (byte)'\n', bytesConsumed,
+                     bytesBuffered - bytesConsumed);
```
Comment (Rick-Anderson) [inline snippets/csharp/pipelines/ProcessLinesAsync.cs]: Left align parms

**Comment by Rick-Anderson** [comment]
@mairaw  asked:

> Let's merge and check?

Yes!

**Comment by mairaw** [comment]
It's rebuilding on master branch now. We can check how it looks on https://review.docs.microsoft.com/en-us/dotnet/standard/io/pipelines?branch=master soon.

**Comment by ozholob** [comment]
@mairaw Let me know if that required some additional updates because I haven't access to the 
https://review.docs.microsoft.com/en-us/dotnet/standard/io/pipelines?branch=master :)

**Comment by Rick-Anderson** [comment]
This is what it looks like in tablet mode (widest browser with no left or right TOC)

image

image

image

image

image

image

image

image

**Pull request merged**